backend/sse/src/server.ts

```
@@ -0,0 +1,197 @@
+
+/**
+ * Implement the streamHiddenTestProcess
+ */
+const redis = new Redis({
+    host: process.env.REDIS_HOST,
+    port: parseInt(process.env.REDIS_PORT || "6379"),
+});
+
+const clients: Map<string, Response> = new Map();
+
+const streamHiddenTestProcess = async (call: any, callback: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)